### ISSUE TITLE
Facing problem after deploying it's not downloading the file it's opening the file

### ISSUE THREAD

**Issue by prasanmoonraft** [dart-lang/web#260]
On click it's opening the file instead of downloading it, wrong suggest here

**Comment by prasanmoonraft** [comment]
@srujzs any idea on this

**Comment by ykmnkmi** [comment]
Did you set [`download`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#download) attribute?

**Comment by prasanmoonraft** [comment]
Yes, tried multiple ways but local it's working after deploying it's opening the file instead of downloading @ykmnkmi

**Comment by srujzs** [comment]
Can you share a minimal repro of what you're attempting?
